src/core/service.h

```diff
@@ -96,6 +96,8 @@ struct Service {
         usec_t restart_usec;
         usec_t timeout_start_usec;
         usec_t timeout_stop_usec;
+        usec_t timeout_abort_usec;
+        usec_t timeout_abort_set;
```

Comment:
Should not the `timeout_abort_set` field be of type bool?

Reply:
Good point. I've created #12311.